### PR TITLE
Show title instead of race

### PR DESCRIPTION
**Pull request by bhollis**
This replaces the race of your character with your chosen title if you have one equipped:

Screen Shot 2022-06-26 at 1 18 22 PM

I need some help to finish this off - the design/colors, and what it should look like when a title is gilded. I couldn't find the gilded shield icons so someone might have to draw it.

### CODE REVIEW

**Comment by bhollis** [comment]
Sadly I have never gilded a seal so my testing ability is quite limited.

**Comment by bhollis** [comment]
OK, I think this is in good shape now! From @robojumper's profile:

image

File: src/app/inventory/store/d2-store-factory.ts [hunk 1]
```diff
@@ -117,3 +127,39 @@ export function getCharacterStatsData(
...
+      profileRecords?.records[titleRecordDef.titleInfo.gildingTrackingRecordHash];
+
+    if (gildedRecord?.completedCount) {
+      gildedNum = gildedRecord.completedCount;
+    }
+
+    isGildedForCurrentSeason = Boolean(
+      gildedRecord && !(gildedRecord.state & DestinyRecordState.ObjectiveNotCompleted)
+    );
+    // todo: figure out whether the title is gilded for the current season
```
Comment (robojumper) [inline src/app/inventory/store/d2-store-factory.ts:161]:
```suggestion
```

**Review by robojumper — APPROVED** [review]

**Review by vivekhnz — APPROVED** [review]
Looks great!

**Pull request merged**